Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/jobs/JobUtils.java

```diff
@@ -153,11 +150,11 @@ public void openDefaultBrowser(@NotNull final URI uri) throws IOException {
         }
     }
 
-    public void openFileExplorer(@NotNull final File file) throws IOException {
+    public static void openFileExplorer(@NotNull final File file) throws IOException {
```

Comment:
Remember the practice? 😄  Calling a static function in a static function, is hard to mock and test.